wire-ios-data-model/Tests/MLS/StaleMLSKeyDetectorTests.swift

```diff
@@ -51,28 +65,34 @@ class StaleMLSKeyDetectorTests: ZMBaseManagedObjectTest {
     }
 
     func test_KeyingMaterialUpdated() throws {
-        try syncMOC.performGroupedAndWait { context in
-            // Given
-            let sut = StaleMLSKeyDetector(
-                refreshIntervalInDays: 5,
-                context: context
-            )
+        var group: MLSGroup!
```

Comment:
would using ? instead be ok? As not crashing the test suite if it does not go as expected

Reply:
I will safely unwrap it, but I'll do it in another PR so to avoid another hour of CI testing.